src/examples/CMakeLists.txt

```diff
@@ -66,3 +66,28 @@ install(
         examples/axom/using-with-blt
 )
 
+
+#------------------------------------------------------------------------------
+# configure and install 'radiuss_tutorial' example
+#------------------------------------------------------------------------------
+configure_file(
+    radiuss_tutorial/host-config.cmake.in
```

Comment:
I think I would just call this the tutorial... but we can debate/fix that later.